modules/shortcodes/dailymotion.php

```diff
@@ -110,12 +131,12 @@ function dailymotion_shortcode( $atts ) {
 	}
 
 	/*set width and height using provided parameters if any */
-	$width = isset($atts['width']) ? abs( $atts['width'] ) : 0 ;
-	$height = isset($atts['height']) ? abs( $atts['height'] ) : 0 ;
+	$width  = isset( $atts['width'] )  ? intval( $atts['width'] )  : 0 ;
+	$height = isset( $atts['height'] ) ? intval( $atts['height'] ) : 0 ;
 
-	if ( ! $width && ! $height) {
+	if ( ! $width && ! $height ) {
 		if ( ! empty( $content_width ) ) {
-			$width = min( 425, intval( $content_width ) );
+			$width = absint( $content_width );
```

Comment:
It will change things a bit, yes, but I think it's best if `$content_width` takes priority when it's provided by the theme and when no `width` is provided in the shortcode.